### PR TITLE
Run pylint issue

### PR DESCRIPTION
**Pull request by sbhatm1213**
Run Pylint step in the CircleCI Build process - has been failing . Trying to fix that.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -74,6 +74,7 @@ jobs:
       - run:
           name: 🧹 Run Rubocop
           command: |
+            bundle install
```
Comment (jbampton) [inline .circleci/config.yml:77]: Is this line needed ?

Comment (sbhatm1213) [reply .circleci/config.yml:77]: I think this is not needed. Should i remove & commit ?

**Comment by BaseMax** [comment]
Thank you @sbhatm1213, good job.

What's your name?

**Review by smriad — DISMISSED** [review]
Looks Good

**Comment by sbhatm1213** [comment]
Hi Max,
My name is Sowjanya . Nice to meet you !
--
Regards
Sowjanya

**Comment by jbampton** [comment]
Can you fix the conflicts ?

**Review by jbampton — APPROVED** [review]

**Comment by BaseMax** [comment]
good job

**Pull request merged**